Makefile

```diff
@@ -156,6 +156,10 @@ deploy: manifests kustomize ## Deploy controller to the K8s cluster specified in
 	cd config/manager && $(KUSTOMIZE) edit set image quay.io/openshift/origin-external-dns-operator=${IMG}
 	# webhook volume and service are added explicilty so that they don't land in the bundle where it's managed by OLM
 	cd config/default && $(KUSTOMIZE) edit add patch --path=manager_webhook_volume_patch.yaml
+	# add service monitor with certificate from the service serving certificate
+	cd config/prometheus && $(KUSTOMIZE) edit add resource monitor.yaml
```

Comment:
Why not add the monitor resource directly or put another way include it in the default target.

Reply:
Done, added `monitor.yaml` as default resource, `tlsConfig` patch is added for the bundle generation only.